02-account-kit-instantiate-usage/index.js

```diff
@@ -0,0 +1,39 @@
+// SDK exposes all you need
+import { Asset, Chains, Int64, UInt64 } from '@[SOMETHING]kit'
+import { Account } from '@[SOMETHING]kit/account'
+
+async function main() {
+    // Loads account representation (optional 2nd parameter for chain, could use a default if not specified)
+    const account: Account = Account.from('teamgreymass', Chains.EOS)
+
+    // Retrieves data from the APIs (v1/chain/get_account)
+    await account.loadData()
```

Comment:
I personally would prefer not having this extra step where you need to load the data manually.

Reply:
I'm not sure its great either - but this was the latter option Johan has mentioned.